src/SFA.DAS.FindEpao.Web/Views/Courses/CourseEpao.cshtml

```diff
@@ -1,48 +1,75 @@
+                    @Model.Epao.Name is 1 of @Model.CourseEpaosCount end-point assessment organisations for @Model.Course.Title <span class="das-no-wrap">(level @Model.Course.Level).</span>
+                </p>
+                <p class="govuk-body govuk-!-margin-bottom-1">
+                    <a class="govuk-link govuk-link--no-visited-state"
+                       asp-route-id="@Model.Course.Id"
+                       asp-route="@RouteNames.CourseEpaos">View the other end-point assessment organisations</a>
+                </p>
+            }
+            else
+            {
+                <p class="govuk-body govuk-!-margin-bottom-1">
+                    @Model.Epao.Name is the only end-point assessment organisation for @Model.Course.Title <span class="das-no-wrap">(level @Model.Course.Level).</span>
+                </p>
+            }
+
             <div class="das-highlight">
                 <p class="govuk-body">
                     @Model.Course.Title <span class="das-no-wrap">(level @Model.Course.Level)</span><br />
-                    
+
                     @if (Model.EffectiveFrom.Date > DateTime.Today)
                     {
-                        <span class="govuk-!-font-weight-bold">Available to assess from @Model.EffectiveFrom.ToString("d MMMM yyyy")</span>
+                        <span class="govuk-!-font-weight-bold">Available to assess from @Model.EffectiveFrom.LongDateFormat()</span>
                     }
                     else
                     {
                         <span class="govuk-!-font-weight-bold">Available to assess now</span>
                     }
                 </p>
-                @if (Model.CourseEpaosCount > 1)
-                {
-                    <p class="govuk-body">
-                        @Model.Epao.Name is 1 of @Model.CourseEpaosCount end-point assessment organisations for @Model.Course.Title <span class="das-no-wrap">(level @Model.Course.Level).</span>
-                    </p>
-                    <p class="govuk-body govuk-!-margin-bottom-1">
-                        <a class="govuk-link govuk-link--no-visited-state"
-                           asp-route-id="@Model.Course.Id"
-                           asp-route="@RouteNames.CourseEpaos">View the other end-point assessment organisations</a>
-                    </p>
-                }
-                else
-                {
-                    <p class="govuk-body govuk-!-margin-bottom-1">
-                        @Model.Epao.Name is the only end-point assessment organisation for @Model.Course.Title <span class="das-no-wrap">(level @Model.Course.Level).</span>
-                    </p>
-                }
+
+                <h2 class="govuk-heading-m" id="StandVerDetails">Standard version details</h2>
+                <table class="govuk-table" aria-describedby="StandVerDetails">
+                    <thead class="govuk-table__header" id="StandardVersionsHead" aria-label="StandardVersionDetails">
```

Comment:
Can we remove the `aria-label="StandardVersionDetails"`? It doesn't do anything, and will be read out by a screenreader

Reply:
removed `aria-label="StandardVersionDetails"`